CHANGELOG.md

```diff
@@ -7,6 +7,21 @@ This document contains a historical list of changes between releases. Only
 changes that impact end-user behavior are listed; changes to documentation or
 internal API changes are not present.
 
+v0.31.1 (2023-02-06)
+--------------------
+
+> **BREAKING CHANGES**: This release has breaking changes. Please read entries
+> carefully and consult the [upgrade guide][] for specific instructions.
+
+### Breaking changes
+
+- All release Windows `.exe` files are now published as a zip archive.
```

Comment:
Sorry I just realised that it's a bit weird how these changes are listed under 0.31.0 in the upgrade guide, but under 0.31.1 in the changelog?

Reply:
Maybe we have to list the breaking changes separately for both v0.31.0 and v0.31.1? I.e. for v0.31.0 we have some binaries unzipped, and for v0.31.1 we have all of them zipped.